src/main/java/at/hannibal2/skyhanni/utils/StringUtils.kt

```diff
@@ -20,11 +20,13 @@ object StringUtils {
     private val resetPattern = "(?i)§R".toPattern()
     private val sFormattingPattern = "(?i)§S".toPattern()
     private val stringColourPattern = "§[0123456789abcdef].*".toPattern()
+    private val asciiPattern = "[^${Char(0)}-${Char(255)}]".toPattern()
```

Comment:
This regex is not for ASCII, ASCII does not go to 255 this regex should be changed to the following. using Char(n) does not provide any benefit here.

```suggestion
private val asciiPattern = "[^\\x00-\\x7F]".toPattern()
```

Reply:
Yeah, made an ups by including 128 characters more 😅. Also I didn't know how to but a specific char into a String in Kotlin, so thank you.